### ISSUE TITLE
[Enhancement] Improve contrast on disabled kirby button to pass WAVE

### ISSUE THREAD

**Issue by esgem** [kirbydesign/designsystem#1005]
*NOT Prioritized*
Contrast for disabled button must change in order to conform to the _Web Accessibility Evaluation Tool_

*NOT UX Refined*
How the contrast should change is most likely a UX decision.

*NOT Tech Refined*
Increasing e.g. the text color to #454545 makes it pass WAVE. 

**Is your enhancement request related to a problem? Please describe.**
When the button is disabled the gray colors fails in the Chrome WAVE plugin (see attached)

**Describe the solution you'd like**
A disabled button conforms to WAVE.

**Describe alternatives you've considered**
none

**Additional context**
![Screenshot with the WAVE output attached](https://user-images.githubusercontent.com/825735/80711116-22de4e80-8af0-11ea-8469-bd6ae75efd4a.png)

**Comment by bommariusser** [comment]
@esgem Thanks for reminding us, but keep in mind WCAG doesn’t require disabled buttons to meet contrast rules (https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) - That doesn't mean that we shouldn't care! :) In general we should be aware of that issue and make sure the UI guides the user to fill out everything before.

Thus I don't think making the grey text darker is a good solution, because this will make it feel active, and then the confusion is real :D

There are a lot of other things we can do though to support better accessibility for disabled buttons, and i'm not sure how much we do, but we already do some things like not being able to focus these buttons... (https://a11y-101.com/development/aria-disabled) etc. etc.  Also in general when talking about tabbing through your forms should leave a inline error when you don't fill them.

I've also seen some user interfaces where a disabled buttons have some text inside or near it to tell the user what they have to do before they can press on it - like this example from wolt, which is quite nice :

<img width="385" alt="Skærmbillede 2020-05-04 kl  20 50 49" src="https://user-images.githubusercontent.com/49033002/81002082-f9535900-8e48-11ea-9d30-eaa6397f4f8d.png">
